docs/INSTALL.md

```diff
@@ -190,21 +190,21 @@ Some tests [are known to be broken](https://github.com/organicmaps/organicmaps/i
 
 Linux, Mac, or Windows should work to build Organic Maps for Android.
 
-Ensure that you have at least 20GB of free space.
+Ensure that you have at least 30GB of free space.
 
 Install [Android Studio](https://developer.android.com/studio).
 
 Install [Android SDK](https://developer.android.com/sdk/index.html) and
 [NDK](https://developer.android.com/tools/sdk/ndk/index.html):
 
 - Run the Android Studio.
-- Open "SDK Manager" ("Tools" → "SDK Manager").
-- Choose "Android 11 (R) API Level 30" SDK.
-- Choose "version "30" and click "OK".
+- Open "SDK Manager" (under "More Actions" in a welcome screen or a three-dot menu in a list of recent projects screen or "Tools" top menu item in an open project).
+- Select "Android 11.0 (R) / API Level 30" SDK.
+- Switch to "SDK Tools" tab
 - Check "Show Package Details" checkbox.
-- Choose "NDK (side by side)" version **21.X.Y**.
-- Choose "CMake" version **3.18.1**.
-- Click "OK".
+- Select "NDK (Side by side)" version **21.X.Y**.
```

Comment:
NDK 21.4.7075529 should be used, as it is defined in build.gradle

Reply:
Will fix that, thanks!
As a side-note: is it a normal practice to keep versions hardcoded like that in build.gradle? Is it possible to move it to gradle.properties?

Reply:
This is the latest NDK that works with our code. There is a complex compilation issue with newer NDKs, so at the moment it is the only working practice :)